admin/preprints/views.py

```diff
@@ -591,6 +591,7 @@ def post(self, request, *args, **kwargs):
 
         from osf.utils.migrations import disable_auto_now_fields
         with disable_auto_now_fields():
+            preprint.requests.all().delete()
```

Comment:
So is there not a middle ground between not deleting any requests and deleting all the requests?